<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>572</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>669</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>669</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>在线订购系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/Icon/Icon/store-48.png</normaloff>:/Icon/Icon/store-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonCart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
    </property>
    <property name="text">
     <string>购物车</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Icon/Icon/cart-white-48.png</normaloff>:/Icon/Icon/cart-white-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>631</width>
      <height>351</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSearch">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>全部</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/Icon/Icon/store-32.png</normaloff>:/Icon/Icon/store-32.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>书籍</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/Icon/Icon/book-32.png</normaloff>:/Icon/Icon/book-32.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>服饰</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/Icon/Icon/cloth-32.png</normaloff>:/Icon/Icon/cloth-32.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>食物</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/Icon/Icon/food-32.png</normaloff>:/Icon/Icon/food-32.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电子产品</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/Icon/Icon/electro-32.png</normaloff>:/Icon/Icon/electro-32.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>90</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Icon/Icon/exit-white-48.png</normaloff>:/Icon/Icon/exit-white-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUserConfig">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
    </property>
    <property name="text">
     <string>用户设置</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Icon/Icon/config-white-48.png</normaloff>:/Icon/Icon/config-white-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(255, 255, 255);border-style: solid</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>671</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(16, 124, 15);border-style: solid</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;color: rgb(16, 124, 15);</string>
    </property>
    <property name="text">
     <string>Store</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>58</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon.qrc">:/Icon/Icon/store-48.png</pixmap>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>buttonCart</zorder>
   <zorder>tableView</zorder>
   <zorder>comboBoxSearch</zorder>
   <zorder>buttonExit</zorder>
   <zorder>buttonUserConfig</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonCart</tabstop>
  <tabstop>comboBoxSearch</tabstop>
  <tabstop>buttonUserConfig</tabstop>
  <tabstop>buttonExit</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
